gapis/trace/android/profile/groups.go

```diff
@@ -107,6 +132,16 @@ func (n *groupTreeNode) visit(parent int32, callback func(parent int32, node *gr
 	}
 }
 
+func (n *groupTreeNode) find(link api.SubCmdIdx) *groupTreeNode {
```

Comment:
Can we give this function a better name? Why does a groupTreeNode have a "find" function that returns a groupTreeNode. It's not finding itself, so what is it finding?

Reply:
Added a comment.